attribute/image.go

```diff
@@ -0,0 +1,52 @@
+package attribute
+
+import (
+	"encoding/base64"
+)
+
+//Deprecated: Will be removed in v3.0.0 - use AttrType instead. ImageType Image format
+type ImageType int
```

Comment:
You seem to be duplicating AttrType?

Reply:
This method was there previously, I had removed it in an earlier commit, before realising the best course of action would be to mark it as deprecated in this release, and then remove it in the next